engine/language-server/src/main/scala/org/enso/languageserver/monitoring/ReadinessMonitor.scala

```diff
@@ -28,7 +29,7 @@ class ReadinessMonitor() extends Actor with ActorLogging {
       context.become(ready())
 
     case InitializationFailed =>
-      log.error("Initialization failed. Terminating JVM...")
+      logger.error("Initialization failed. Terminating JVM...")
       System.exit(1)
```

Comment:
While this is not related to the change, I only now noticed it and have a feeling that this message may not be logged. As the underlying logger is using a message queue on a different thread, it may take a short moment before it is actually written and it may not do it on time before `System.exit` terminates the application.